src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -3953,7 +3953,7 @@ protected function addSetPrimaryKeyMultiPK(string &$script): void
      * @param array \$keys The elements of the composite key (order must match the order in XML file).
```

Comment:
can we specify here the value type? array<..> etc?

Reply:
Looks like for this case we can have an array  with mixed value 
If we have primary key use multiple-column index

```
CREATE TABLE
    item (
        tag VARCHAR(255) NOT NULL,
        id INT NOT NULL,
        PRIMARY KEY(tag, id)
    );
```